sdk/core/core-tracing/src/interfaces.ts

```diff
@@ -115,96 +105,6 @@ export enum SpanKind {
   CONSUMER = 4
 }
 
-/**
- * An Exception for a Span.
- */
-export declare type Exception =
-  | ExceptionWithCode
-  | ExceptionWithMessage
-  | ExceptionWithName
-  | string;
-
-/**
- * An Exception with a code.
- */
-export interface ExceptionWithCode {
-  /** The code. */
-  code: string | number;
-  /** The name. */
-  name?: string;
-  /** The message. */
-  message?: string;
-  /** The stack. */
-  stack?: string;
-}
-
-/**
- * An Exception with a message.
- */
-export interface ExceptionWithMessage {
-  /** The code. */
-  code?: string | number;
-  /** The message. */
-  message: string;
-  /** The name. */
-  name?: string;
-  /** The stack. */
-  stack?: string;
-}
-
-/**
- * An Exception with a name.
- */
-export interface ExceptionWithName {
-  /** The code. */
-  code?: string | number;
-  /** The message. */
-  message?: string;
-  /** The name. */
-  name: string;
-  /** The stack. */
-  stack?: string;
-}
-
-/**
- * Return the span if one exists
- *
- * @param context - context to get span from
- */
-export function getSpan(context: Context): Span | undefined {
-  return otTrace.getSpan(context);
-}
-
-/**
- * Set the span on a context
- *
- * @param context - context to use as parent
- * @param span - span to set active
- */
-export function setSpan(context: Context, span: Span): Context {
```

Comment:
This (and setSpanContext) are being used [in servicebus](https://github.com/Azure/azure-sdk-for-js/blob/45de594e60f61d352d4467066a2c22223fdf6f98/sdk/servicebus/service-bus/src/diagnostics/tracing.ts#L227-L228) and [eventhubs](https://github.com/Azure/azure-sdk-for-js/blob/45de594e60f61d352d4467066a2c22223fdf6f98/sdk/eventhub/event-hubs/src/diagnostics/tracing.ts#L121-L122) 

I talked to @chradek about it, one option is they take a dependency on OpenTelemetry directly. Another is we keep these two functions here for the time being